Index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="Style.css">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>HTML-CSS-forms</title>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <script src="https://kit.fontawesome.com/5036f34c07.js" crossorigin="anonymous"></script> 
+    </head>
+
+<body>
+
+    <header>
+        <nav class="navb">
+            <div class="sign-in">
+                <div class="text">
+                <small>Already have an Intuit account?</small> </br>
+                <small style="color: blue">I forgot my User ID or Password</small>
```

Comment:
- [ ] Do not use inline styling. Rather, implement this on your CSS file to keep everything organized.